docs/backstage.md

```diff
@@ -0,0 +1,124 @@
+    ```
+
+4. **Optional - Run the Application Locally**: After building the project, you can run it locally. Ensure that all necessary environment variables are set:
+
+    ```sh
+    export BASE_URL=https://your-local-base-url.com
+    export POSTGRES_HOST=your-local-postgres-host
+    export POSTGRES_PORT=your-local-postgres-port
+    export POSTGRES_USER=your-local-postgres-user
+    export POSTGRES_PASSWORD=your-local-postgres-password
+    export POSTGRES_DB=your-local-postgres-db
+
+    yarn start
+    ```
+
+5. **Ensure Azure CLI is Installed**: Make sure you have the Azure CLI installed and logged in. If not, install it from [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) and log in using:
+
+    ```sh
+    az login
+    ```
+
+6. **Set Environment Variables**: Set the `ACR_NAME` and `RESOURCE_GROUP` environment variables:
+
+    ```sh
+    export ACR_NAME=your_acr_name
+    export RESOURCE_GROUP=your_resource_group
+    ```
+
+7. **Build the Docker Image Locally**: Use the `docker build` command to build your Docker image:
+
+    ```sh
+    docker build -t $ACR_NAME.azurecr.io/my-backend-app:latest .
+    ```
+
+8. **Login to Azure Container Registry**: Use the Azure CLI to log in to your ACR:
+
+    ```sh
+    az acr login --name $ACR_NAME --resource-group $RESOURCE_GROUP
+    ```
+
+9. **Push the Docker Image to ACR**: Push the built image to your ACR:
+
+    ```sh
+    docker push $ACR_NAME.azurecr.io/my-backend-app:latest
+    ```
+
+10. **Verify the Image in ACR**: You can verify that the image has been pushed to ACR by listing the repositories:
+
+    ```sh
+    az acr repository list --name $ACR_NAME --output table
+    ```
+
+
+## Getting Started
+
+  To get started with Backstage in this project, follow these steps:
+
+1. **Fork & Clone the Repository**:
+    - First, fork the repository to your own GitHub account by clicking the "Fork" button on the repository page.
+    - Then, clone your forked repository:
+    ```sh
+    git clone https://github.com/<your_fork>/aks-platform-engineering.git
+    cd aks-platform-engineering
+    ```
+
+2. **Deploy Terraform with Backstage**:
+    To deploy Backstage, you can use the provided Terraform scripts. Navigate to the `terraform` directory and apply the configuration:
+    ```sh
+    cd terraform
+    terraform apply -var infrastructure_provider=crossplane -var build_backstage=true -var gitops_addons_org=https://github.com/<your_gh_username/org> -var github_token=<your_pat_token> --auto-approve
+    ```
+
+    > **Note:** There is an alert generated in the terraform output that informs you to go to your newly created backstage SP and "Grant Admin" to the SP. This is because onboarding all users from your entra tenant requires admin privileges. If you don't do this prior to Backstage deploying you will have to wait for the scheduled task to run again (Once an hour) or manually populate the users in the backstage user table. 
+
+
+## Additional Resources
+
```

Comment:
Would be nice to have a FAQ/Troubleshooting section.  Add this which I hit.

If this error occurs on `terraform apply`:
`│ ApplicationsClient.BaseClient.Post(): unexpected status 403 with OData error: Authorization_RequestDenied: Insufficient privileges to complete the operation.`
[give Application.ReadWrite.All permissions](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/guides/service_principal_configuration#method-1-api-roles-recommended-for-service-principals).